13.go

```diff
@@ -868,16 +869,24 @@ func verifyPeerHandshakeSignature(
 }
 
 func (hs *clientHandshakeState) getCertificate13(certReq *certificateRequestMsg13) (*Certificate, error) {
+	// Prefer sign algorithms from supported_algorithms_cert extension if available
+	var sigAlgoCert []SignatureScheme
```

Comment:
Remove this, since this is no longer done on the server side.

Reply:
yes, actually I don't want whole this change anymore